soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -570,7 +570,7 @@ static void PlaceVanillaCowMilk() {
 }
 
 static void SetScarceItemPool() {
-  ReplaceMaxItem(PROGRESSIVE_BOMBCHUS, 3);
+  ReplaceMaxItem(PROGRESSIVE_BOMBCHUS, InfiniteUpgrades ? 2 : 3);//reduced with infinite upgrades because the second one gives infinite
```

Comment:
Same here, I think you'd still want 3 _chances_ to find bombchus, and your infinite upgrade would just be a _bonus_ after finding the second (or subsequent) ones like it is without this change.  It's just that the bonus now is that you get INFINITELY more of them instead of X more of them.

Reply:
Same as the other one.